packages/atlas/atlas.config.yml

```diff
@@ -24,7 +24,7 @@ storage:
 
 joystream:
   tokenTicker: 'JOY' # Ticker for the token used in the app
-  tokenPriceFeedUrl: null # URL for the token price feed - used to display token price in the app
+  tokenPriceFeedUrl: 'https://status.joystream.org/price' # URL for the token price feed - used to display token price in the app
```

Comment:
if you set `tokenPriceFeedUrl` to null we shouldn't show the price in dollars at all. Now we're showing 0 in a lot of places
![image](https://github.com/Joystream/atlas/assets/51168865/dd79bf69-62cc-4d2c-a108-4b3bec35db82)
![image](https://github.com/Joystream/atlas/assets/51168865/65dac87f-a371-4391-87ea-46959a8ea560)
![image](https://github.com/Joystream/atlas/assets/51168865/555b7389-fd7b-4376-a5c8-e4edcc201c20)